typed/racket/stream.rkt

```diff
@@ -19,3 +19,7 @@
  [stream-append (All (a) [(Sequenceof a) * -> (Sequenceof a)])]
  )
 
+(require/typed/provide
+ racket
```

Comment:
It would be better to require it from `racket/base`, avoid requiring all of `racket`.

Reply:
Sure!

I was under the impression that `sequence->stream` was not provided by `racket/base`, but you're clearly right.